Tasks/UsePythonVersion/Strings/resources.resjson/en-US/resources.resjson

```diff
@@ -0,0 +1,15 @@
+{
+  "loc.friendlyName": "Use Python Version",
+  "loc.helpMarkDown": "",
+  "loc.description": "Retrieves the specified version of Python from the tool cache. Optionally add it to PATH.",
+  "loc.instanceNameFormat": "Use Python $(versionSpec)",
+  "loc.input.label.versionSpec": "Version spec",
+  "loc.input.help.versionSpec": "Version range or exact version of a Python version you want to use.",
```

Comment:
What do you think about adding some example text here?  Such as something like: `(e.g. '>= 3.3', or '2.7')`

Reply:
Maybe -- this is the same version range language we use in all of our tool tasks (such as in the Node Tool Installer).  It is the same version language used by NPM.  I felt that giving setting `>= 3.3` as the default should give users the gist, and we can lean on the documentation for a more thorough explanation.

My guess is users will almost always use major.minor version specs.